src/main/kotlin/no/nav/klage/oppgave/clients/azure/MicrosoftGraphClient.kt

```diff
@@ -97,11 +97,7 @@ class MicrosoftGraphClient(
             it.value ?: emptyList()
         }.mapNotNull {
             secureLogger.debug("Display name: $it")
-            if (it.onPremisesSamAccountName == null || it.displayName == null) {
-                null
-            } else {
-                it.onPremisesSamAccountName to it.displayName
-            }
+            it.onPremisesSamAccountName to it.displayName
```

Comment:
Kan dette ikke lenger skje? Eller kunne det aldri?

Reply:
Feltene er iallefall ikke nullable lenger. Det er en visst mulighet for at jeg har innført en bug selvsagt, men jeg har også fått ryddet litt i både kode og Azure-rettighetene appen vår har, så jeg håper det skal fungere.